clarifai/cli/compute_cluster.py

```diff
@@ -0,0 +1,52 @@
+import click
+from clarifai.cli.base import cli
+from clarifai.client.user import User
+from clarifai.utils.cli import display_co_resources
+
+
+@cli.group()
+@click.pass_context
+def computecluster(ctx):
+  """Manage Compute Clusters: create, delete, list"""
+  ctx.obj = User(user_id=ctx.obj['user_id'], pat=ctx.obj['pat'], base_url=ctx.obj['base_url'])
```

Comment:
oh I realised, `ctx.obj['user_id']` could be `None` if it not set in either as env var or during  clarifai login, so need to handle that condition too.

Reply:
Added a prompt similar to `pat`